### DISCUSSION TITLE
Broken links in Handbook

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1231] [Q&A] [answered]
Hey! Just noticed that the Mox Wiki (linked to in the handbook's Computing Resources page) isn't working!

https://robertslab.github.io/resources/mox_Adding-a-User/
https://robertslab.github.io/mox_Logging-In.html

And so on all either end in 404s or otherwise broken pages.

Not a big deal since most of the info also exists at [the UW wiki](https://wiki.cac.washington.edu/display/hyakusers/Hyak+mox+Overview), but just thought I'd let someone  know!

**Comment by sr320** [comment]
@afcoyle can you be more specific where you are finding these links?

I do not find errors in handbook https://d.pr/i/UoFbjV but agree links you provide are bad

**Reply by kubu4** [reply]
Sometimes, I don't know what happens, but I read something and then write almost exactly the same thing. When I typed my reply below, I totally misread/interpreted what @sr320 wrote.

So, I've contributed nothing to this discussion, except trying to steal credit for an answer from someone else. Sorry!

**Comment by kubu4** [comment]
To riff on a famous Star Wars quote: 

Those aren't the links you're looking for...

You want:

https://robertslab.github.io/resources/mox_Adding-a-User.html

https://robertslab.github.io/resources/mox_Logging-In.html

I'm not sure where you grabbed those original URLs from.

**Comment by aspencoyle** [comment, marked as answer]
Hmm, looks like the discrepancy is the sidebar usage! I was following this path:

Starting at https://robertslab.github.io/resources/

Scrolling down to "Sticking to the Keyboard?" and clicking the "Computing Hardware" link

Scrolling down to "Mox (Hyak)" and clicking the "Mox Wiki" link. 

Hopefully that's enough info for you to find them!

**Reply by kubu4** [reply]
Thanks for the explanation. Fixed them!
